blockchain/v2/scheduler.go

```diff
@@ -111,15 +197,15 @@ func newSchedule(initHeight int64) *schedule {
 	return &sc
 }
 
-func (sc *schedule) addPeer(peerID p2p.ID) error {
+func (sc *scheduler) addPeer(peerID p2p.ID) error {
 	if _, ok := sc.peers[peerID]; ok {
 		return fmt.Errorf("Cannot add duplicate peer %s", peerID)
```

Comment:
```suggestion
		return fmt.Errorf("cannot add duplicate peer %s", peerID)
```
Change to lower case everywhere.